daemon/src/taker_cfd_actor.rs

```diff
@@ -17,11 +17,17 @@ use tokio::sync::{mpsc, watch};
 #[allow(clippy::large_enum_variant)]
 pub enum Command {
     SyncWallet,
-    TakeOrder { order_id: OrderId, quantity: Usd },
+    TakeOrder {
+        order_id: OrderId,
+        quantity: Usd,
+    },
```

Comment:
Reformatting? 🤔

Reply:
Yeah I meant whether or not this is really just reformatting and why it is happening :D